### DISCUSSION TITLE
Extent der Übersichtskarte

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#82] [Q&A]
Gibt es eine Möglichkeit den Extent der Übersichtskarte sowohl im Kartenbild als auch beim Ausdruck einzustellen?

Bei uns schaut das derzeit nicht gerade "prickelnd" aus, 

Übersichtskarte Kartenbild:

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/e001fa57-4476-4391-a2cd-f68e52ce8a22)

Übersichtskarte Ausdruck:

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/5c80ed92-463e-4252-aba8-869920203de1)

**Comment by gis-eni** [comment]
Zum Viewer:
Für die Übersichtskarte wir das Leaflet Plugin "MiniMap" verwendet: https://github.com/Norkart/Leaflet-MiniMap
Es gibt hier verschiedene Optionen die man der Übersichtskarte mitgeben kann (siehe Github Repo). Du kannst die MiniMap Optionen in der ``custom.js`` selber setzten. Das hier ist standardmäßig gesetzt:

``` javascript
 webgis.usability.miniMapOptions: {
     zoomLevelOffset: -5,
     position: 'bottomleft',
     toggleDisplay: true,
     minimized: true
 }
``` 
Wie weit beim Start hinausgezoomt wird, hängt von deiner Kartenausdehnung ab, die du im CMS für die Karte definiert hast. Der ``zoomLevelOffset`` ist per default ``-5``. Dh. immer um 5 Levels über dem aktuellen Zoom.

Zum Druck:
Hier wird als Extent auch das verwendet, was du in der Kartenausdehnung im CMS definiert hast. Außerdem kannst du auch die Größe der Übersichtkarte im Drucklayout anpassen.
